source/import-javascript/index.html.md.erb

```diff
@@ -65,23 +65,30 @@ If your service cannot use ES modules, read about [alternative module formats](#
 
 ### Import individual components
```

Comment:
I've updated the 'Configure components' to talk about 'specific components' rather than 'individual components' as we're now instanciating all instances of a given component rather than a single component (like we had in some of our previous examples). Should we align here as well?